lte/gateway/c/core/oai/tasks/amf/amf_authentication.cpp

```diff
@@ -122,48 +120,76 @@ static int start_authentication_information_procedure(
   auth_info_proc->ue_id = ue_id;
   auth_info_proc->resync = auth_info_proc->request_sent;
 
-  bool is_initial_req = !(auth_info_proc->request_sent);
-  auth_info_proc->request_sent = true;
+  rc = amf_authentication_request_sent(ue_id);
+  if (rc != RETURNok) {
+    OAILOG_FUNC_RETURN(LOG_NAS_AMF, rc);
```

Comment:
Better to add a log here along with IMSI/ue_id. Helps in debugging

Reply:
comment addressed.